site_scons/components/__init__.py

```diff
@@ -325,7 +325,9 @@ def define_components(reqs):
                           'cp -r -P dpdk/build/include/* '                     \
                           '"$SPDK_PREFIX/include/dpdk"',
                           'mkdir -p "$SPDK_PREFIX/share/spdk"',
-                          'cp -r include scripts "$SPDK_PREFIX/share/spdk"'],
+                          'cp -r include scripts "$SPDK_PREFIX/share/spdk"',
+                          'cp build/examples/{lsvmd,nvme_manage} '             \
```

Comment:
what about 'identify' and 'perf'? should we add them as well?

Reply:
I don't have a particular opinion so I will add them.